src/cdi_analytic/Pseudo_Line_Base.cc

```diff
@@ -249,7 +249,8 @@ double Pseudo_Line_Base::monoOpacity(double const x, double const T) const {
       Result += edge_factor_[i] * cube(nu0 / x);
     }
   }
-  if (Tpow_ != 0.0) {
+  if (!rtt_dsxx::soft_equiv(Tpow_, 0.0,
+                            std::numeric_limits<double>::epsilon())) {
```

Comment:
The value reported by numeric_limits is more representative of the relative tolerance between doubles (I think for Intel Xeon it is 2.2e-16). `epsilon` is also based on the underlying hardware, so it will likely have a different value on Sierra, which uses IBM Power9 chips.